delivery_api_client/api/delivery_api.py

```diff
@@ -68,35 +76,61 @@ def execute_with_http_info(self, ims_org_id, session_id, delivery_request, **kwa
+        :type session_id: str
+        :param delivery_request: Delivery Request (required)
+        :type delivery_request: DeliveryRequest
+        :param version: This is the `version` of at.js.
+        :type version: str
+        :param async_req: Whether to execute the request asynchronously.
+        :type async_req: bool, optional
         :param _return_http_data_only: response data without head status code
                                        and headers
+        :type _return_http_data_only: bool, optional
         :param _preload_content: if False, the urllib3.HTTPResponse object will
                                  be returned without reading/decoding response
                                  data. Default is True.
+        :type _preload_content: bool, optional
         :param _request_timeout: timeout setting for this request. If one
                                  number provided, it will be total request
                                  timeout. It can also be a pair (tuple) of
                                  (connection, read) timeouts.
-        :return: tuple(DeliveryResponse, status_code(int), headers(HTTPHeaderDict))
+        :param _request_auth: set to override the auth_settings for an a single
+                              request; this effectively ignores the authentication
+                              in the spec for a single request.
+        :type _request_auth: dict, optional
+        :return: Returns the result object.
                  If the method is called asynchronously,
                  returns the request thread.
+        :rtype: tuple(DeliveryResponse, status_code(int), headers(HTTPHeaderDict))
         """
 
         local_var_params = locals()
 
-        all_params = ['ims_org_id', 'session_id', 'delivery_request', 'version']  # noqa: E501
-        all_params.append('async_req')
-        all_params.append('_return_http_data_only')
-        all_params.append('_preload_content')
-        all_params.append('_request_timeout')
+        all_params = [
+            'ims_org_id',
+            'session_id',
+            'delivery_request',
+            'version'
+        ]
+        all_params.extend(
```

Comment:
Can this be one statement [a, b, c, ...]?  Curious why we have to call extend?

Reply:
Everything under the /delivery_api_client directory is generated by the openapi generator tool.  So I don't modify anything in there.